docs/setup-and-ide-alternatives.md

```diff
@@ -7,6 +7,7 @@ The [supported platforms](https://swift.org/platform-support/) for running Swift
 A number of editors you may already be familiar with have support for writing Swift code. Here we provide a non-exhaustive list of such editors and relevant plugins/extensions, sorted alphabetically.
 
 * [Atom IDE support](https://atom.io/packages/ide-swift)
+    * [Atomic Blonde](https://atom.io/packages/atomic-blonde) a SourceKit based syntax highlighter for Atom.
```

Comment:
these changes seem unrelated?

Reply:
These seem to be related to this commit (not mine): https://github.com/swift-server/guides/commit/00340e7157bb1edf5339e1e8fc9091998a64ff03